client/one-time-payment/react/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "react",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```

If possible, let's have this be `start` to match our conventions in the other examples to keep it easy and consistent.
```suggestion
    "start": "vite",
```